.github/workflows/ci.yml

```diff
@@ -20,15 +20,10 @@ jobs:
     - name: Run tests (BouncyCastle)
       run: |
         dotnet test tests/DotNetLightning.Core.Tests -p:BouncyCastle=True
+        dotnet test tests/DotNetLightning.Infrastructure.Tests -p:BouncyCastle=True
```

Comment:
not sure this is going to work as is because AFAIR the infrastructure tests need .NETCore3.0 to be compiled, that's why I was recommending to move this step after the whole solution gets compiled (and keep the `dotnet pack` call before .NETCore3 is installed, to make sure DNL.Core is still .NETCore2x compatible).

Reply:
actually, NETCore 3 is apparently installed before the test run starts. So should I uninstall NetCore 3 after doing native build and move BouncyCastle down below?

Reply:
no, .NETCore3 is installed later, so I'm not sure how this has worked

Reply:
is it because the fucking GitHub guys have decide to start installing .NETCore3 by default on the ubuntu VMs???

Reply:
Yes, as you can see in [the docs](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/software-installed-on-github-hosted-runners#ubuntu-1804-lts) it is installed in Ubuntu 18.04.

Reply:
FFS... oh well, is there an easy way to uninstall it before installing .NETCore2? with apt maybe?

Reply:
Yes, that is what I asked you: 

> So should I uninstall NetCore 3 after doing native build and move BouncyCastle down below?

Reply:
ah great yes let's do that

Reply:
and nit: if you edit the PR description, commit msg should follow too :)